django_ses_plus/settings.py

```diff
@@ -3,7 +3,9 @@
 DJANGO_SES_PLUS_SETTINGS = getattr(settings, "DJANGO_SES_PLUS_SETTINGS", {})
 
 DJANGO_SES_PLUS_SETTINGS.setdefault("SEND_EMAIL", True)
-DJANGO_SES_PLUS_SETTINGS["CELERY_TASK_RETRY_KWARGS"].setdefault("CELERY_TASK_RETRY_KWARGS", {'max_retries': 15, 'countdown': 60})
 
-# Get default from email from django settings.
-DJANGO_SES_PLUS_SETTINGS["DEFAULT_FROM_EMAIL"] = settings.DEFAULT_FROM_EMAIL
+if DJANGO_SES_PLUS_SETTINGS["SEND_EMAIL"]:
```

Comment:
You are defining default settings so you don't need to check if send email is true.  This can be overrode during tests so other default variables should be available.

Reply:
I guess I got the point. I have set defaults for all settings, so user won't get error if he changes settings during tests.

Reply:
Thank you, looks good to me.